src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -69,10 +69,16 @@
   };
   private static final ValueMap EMPTY = new ValueMap(Collections.<String, Object>emptyMap());
   private static final String DEFAULT_CATEGORY = "All";
+  private static final String ENHANCED_ECOMMERCE = "EnhancedEcommerce";
   static final Pattern COMPLETED_ORDER_PATTERN =
```

Comment:
Should we also support [order completed ](https://segment.com/docs/spec/ecommerce/v2/#order-completed) here?